endpoint/endpoint.go

```diff
@@ -169,6 +169,11 @@ func (e *Endpoint) WithSetIdentifier(setIdentifier string) *Endpoint {
 	return e
 }
 
+// EndpointsHaveEmptyTargets checks if an endpoint has no targets
+func EndpointsHaveEmptyTargets(ingEndpoints []*Endpoint) bool {
+	return len(ingEndpoints) == 1 && len(ingEndpoints[0].Targets) == 0
```

Comment:
what if more than one endpoint?

Reply:
This function is targeted to finding specifically the fake endpoint we fabricate only to be skipped later and this case produces a list of exactly one endpoint with an empty list of targets.

Reply:
I was wrong. As a matter of fact, it'll produce several endpoints. We might need to rethink this part.

Reply:
Fixed the check in b0254ad52b8621a99af078bc5ac1cc0f9864154d